<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartNewProduct</class>
 <widget class="QDialog" name="StartNewProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Product</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Customer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cboCustomer"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Product Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtProductName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Product Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboSFK">
       <item>
        <property name="text">
         <string>SFK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 Ply</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 Ply</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Die Cut Carton 3 Ply</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Die Cut Carton 5 Ply</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Die Cut Carton 3 Ply (H)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Die Cut Carton 5 Ply (H)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>StartNewProduct</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
